src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
+    String blabla;
```

Comment:
The variable 'blabla' should have proper naming to enhance code readability. (Severity: low)